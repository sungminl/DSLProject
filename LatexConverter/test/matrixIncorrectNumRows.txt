MATRIX
ROW:3,COL:3
{
1, 2, 3|
4, 5, 6|
}