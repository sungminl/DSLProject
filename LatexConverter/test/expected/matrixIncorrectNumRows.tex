The program could not successfully complete.
A runtime error was encountered:
Number of rows in matrix content does not match declared number of rows.
